### PR TITLE
ITL-30 Change style to use tailwind CSS classes

### PR DESCRIPTION
**Pull request by bgellerth**

### CODE REVIEW

File: src/assets/main.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        width: 600px;
+    }
+    .min-h-82{
+        min-height: 82px;
+    }
+
+    .min-h-163{
+        min-height: 163px;
+    }
+.dropdown-toggle{
```
Comment (kincsokiss) [inline src/assets/main.css]: Please format the content.

File: src/components/Header.vue [hunk 2]
```diff
@@ -1,69 +1,25 @@
...
+      </button>
     </div>
   </div>
 </template>
 
 <script setup lang="ts">
-
 const emit = defineEmits<{
-  (e: 'todoAdded'): void;
+  (e: "todoAdded"): void;
```
Comment (kincsokiss) [inline src/components/Header.vue]: Please check your editor settings and set to use single quote instead of double.

**Review by kincsokiss — CHANGES_REQUESTED** [review]

File: src/assets/main.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    }
+    .min-h-82{
+        min-height: 82px;
+    }
+
+    .min-h-163{
+        min-height: 163px;
+    }
+    .dropdown-toggle{
+    width: 125px;
```
Comment (kincsokiss) [inline src/assets/main.css:16]: Please format this part in the same way how the other classes are and remove the empty line.

**Review by kincsokiss — CHANGES_REQUESTED** [review]

File: src/components/PrioSelector.vue [hunk 4]
```diff
@@ -2,49 +2,98 @@
...
+          toDo.priorityChange,
+      },
+    ]"
   >
     <div
       @click.stop="selectPriority(toDo, 0)"
-      :class="['low', 'priorities', { selected: toDo.priority === 0 }]"
+      :class="[
+        'bg-teal-400',
+        'h-3 w-3 rounded-full cursor-pointer md:!bg-white md:h-8 md:w-full md:rounded-full md:text-xs md:p-0',
```
Comment (gerebdavid) [inline src/components/PrioSelector.vue]: Please create a class for this row, you are using it 3 times.

File: src/components/TodoForm.vue [hunk 5]
```diff
@@ -24,35 +27,37 @@
...
-      :checkedTodos="checkedTodos"
-      @notCheckedTodos="notCheckedTodos"
-      @toggleEdit="toggleEdit"
-      @saveTodo="saveTodo"
-      @deleteCheckedTodos="deleteCheckedTodos"
-      @editChecked="editChecked"
-    />
+    <div class="w-full flex items-center flex-col">
+      <TodoChecked
+        v-if="checkedTodos.length > 0"
```
Comment (gerebdavid) [inline src/components/TodoForm.vue]: checkedTodos.length is enough for the condition

**Review by gerebdavid — APPROVED** [review]

**Pull request merged**